src/npc.cpp

```diff
@@ -1053,7 +1052,7 @@ int NpcScriptInterface::luaNpcCloseShopWindow(lua_State* L)
 	int32_t buyCallback;
 	int32_t sellCallback;
 
-	Npc* merchant = player->getShopOwner(buyCallback, sellCallback);
+	auto merchant = player->getShopOwner(buyCallback, sellCallback);
```

Comment:
Could this be `const auto`
```suggestion
	const auto merchant = player->getShopOwner(buyCallback, sellCallback);
```